### DISCUSSION TITLE
UX major issues with how user's changes in bookmarked view are silently dropped, while root view retains user changes

### DISCUSSION THREAD

**Discussion by bamboowonder** [directus/directus#18672] [Feature Requests]
### Summary

This is a request that Bookmarked views retain user specific changes without making a new bookmark (similar to the root view behavior)

We make extensive use of bookmarks.  Our users are technically ...challenged...

It has been a huge footgun rolling out Directus, that the behavior between a root collection, and it's nested bookmarks are so different.

It is a HUGE plus that Directus retains user specific changes to collection filters/columns/sorts, so when a user navigates back to a collection their changes are silently retained.

However, if they are in a bookmarked view for that collection, not only are any changes to filter/columns/sorts not saved, but they are silently dropped with no warning to the users.

There is no visual indication to indicate that a root collection is any different than a bookmarked view.

Navigating away to edit a record within the bookmarked view, and then returning to the collection's bookmarked view results in losing all user specific refinements to that view.

Creating yet another saved bookmark for every list refinement of filters is a huge pain.

### Motivation

Quality of life for end-users that Directus not silently drop filter/sort/column changes, when it so readily retains those changes in the totally indistinguishable root collection.

### Detailed Design

Request for consistency between bookmarked views and the user specific changes that are automatically saved for root collections.

### Must Have:
bookmarked views also retain a user-specific automatically saved version, similar to how the root collection behaves.

### Drawbacks

There is no reason I can see not to solve this end-user footgun.

### Alternatives

proper support for database views?  but not really.  this is a UI issue.

### Adoption Strategy

it could potentially be a feature toggle, if for some reason end users enjoy having their settings silently dropped and want to retain that footgun functionality.

**Comment by bamboowonder** [comment]
I'm bumping this, as it is just such a horrible user experience, to silently lose any filters/sorts/searches on views, especially when base tables have treated them to the amazing experience of having their incremental filter/sort/searches retained when navigating back.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
